ansible/playbooks/roles/encoder-network/tasks/main.yaml

```
@@ -0,0 +1,15 @@
+---
+- name: Ensure nftables is installed
+  apt:
+    package: nftables
+    state: latest
+
+- name: Create nftables config file
+  copy:
```

Let's make this a template, so we can make it more flexible.

```suggestion
  template:
```